docs/data/charts/tooltip/tooltip.md

```diff
@@ -90,141 +92,151 @@ It removes the header showing the x-axis value from the tooltip.
 
-### Overriding placement
+## Creating a tooltip
 
-To override tooltip placement, override to the tooltip with `slots.tooltip`.
-If you want to keep the default content, you can place the `ChartsItemTooltipContent` or `ChartsAxisTooltipContent` in your custom tooltip.
+To create your custom tooltip, the library exports some helpers which are explained in the following sections:
 
-### Styling
+- `<ChartsTooltipContainer />` A wrapper providing the open/close state and the position of the tooltip.
+- `<ChartsItemTooltipContent />` render the content of the default item tooltip.
+- `<ChartsAxisTooltipContent />` render the content of the default axis tooltip.
+- `useItemTooltip()` provides all basic information associated to the current item tooltip.
+- `useAxesTooltip()` provides all basic information associated to the current axes tooltip.
 
-The tooltip can be styled using CSS classes, similar to other elements. However, there is one caveat.
+### Modifying the position
 
-:::warning
-
-By default, the tooltip is rendered as a child of the document's body, so using the chart's `sx` prop does not work.
-
-```tsx
-import { chartsTooltipClasses } from '@mui/x-charts';
+To override the tooltip position, you can create a wrapper that manages the position.
 
-<LineChart
-  sx={{
-    [`& .${chartsTooltipClasses.root} .${chartsTooltipClasses.valueCell}`]: {
-      color: 'red',
-    },
-  }}
-/>;
+```jsx
+function CustomTooltipPopper(props){
+  // ... (event management) ...
+
+  return <NoSsr>
+      <Popper {/* position */}>
+        {props.children}
+      </Popper>
+    </NoSsr>
+}
 ```
 
-:::
+Then you can either render built-in content (with `<ChartsItemTooltipContent />` or `<ChartsAxisTooltipContent />`) or your own component.
 
-To apply the same style as we're trying to apply above, we need to use the `sx` property inside `slotProps.tooltip`:
+```jsx
+<CustomTooltipPopper>
+  <ChartsItemTooltipContent />
+</CustomTooltipPopper>
+```
 
-{{"demo": "TooltipStyle.js"}}
+The following demo shows example about how to use additional hooks such as `useXAxis()` or `useDrawingArea()` to customize the tooltip position.
 
-You can also disable the portal by setting `slotProps.tooltip.disablePortal` to `true`.
+{{"demo": "CustomTooltipPosition.js"}}
 
-## Composition
+#### Modifying the content
 
-If you're using composition, by default, the axis listens for mouse events to get its current x/y values.
-If you don't need it, you can disable those listeners with the `disableAxisListener` prop.
+To keep the default placement, use the `<ChartsTooltipContainer />` wrapper.
+It accept a prop `trigger = 'item' | 'axis'` to know when the Popper should be open.
 
-You need those listeners if you are using [axes highlight](/x/react-charts/highlighting/#highlighting-axis) or you have a tooltip [triggered by axis](/x/react-charts/tooltip/#tooltip-trigger).
+:::warning
+Do not skip rendering the ChartsTooltipContainer.
+For example the following code does not work.
```

Comment:
Should we briefly explain why?

Reply:
```suggestion
Do not skip rendering the ChartsTooltipContainer.
Otherwise, it could miss some events triggering its opening.

For example the following code does not work.
```

Initially it's because we use the `pointerEnter` event to know if the tooltip is on desktop or mobile. But we could probably modify that and remove this warning

Reply:
Makes sense. Here's my suggestion, but yours work too.

```suggestion
The tooltip might not open if the ChartsTooltipContainer isn't rendered. 

For example the following code does not work.
```

> But we could probably modify that and remove this warning

Yeah, that would be nice.